Assets/_Laz/Scripts/Tools/Writer.cs

```diff
@@ -0,0 +1,30 @@
+using System.IO;
+using System.Text;
+using UnityEngine;
+
+public static class Writer
```

Comment:
Depending on how often we see ourselves Serializing Data into Json files in the future, I might suggest adding this to the PerigonTools Package since it doesn't have any functionality strictly dependant on Laz

Reply:
That's possible, But you'll need to create an interface for this.

Reply:
Definitely out of scope for this pull request, but we could look into also creating a wizard for creating codegen DTO classes and that might help make this a more fleshed out tool